codec-haproxy/src/main/java/io/netty/handler/codec/haproxy/HAProxyMessage.java

```diff
@@ -326,50 +430,52 @@ private static int portStringToInt(String value) {
     /**
      * Validate an address (IPv4, IPv6, Unix Socket)
      *
-     * @param address                    human-readable address
-     * @param addrFamily                 the {@link AddressFamily} to check the address against
-     * @throws HAProxyProtocolException  if the address is invalid
+     * @param address human-readable address
+     * @param addrFamily the {@link AddressFamily} to check the address against
+     *
+     * @throws HAProxyProtocolException if the address is invalid
      */
     private static void checkAddress(String address, AddressFamily addrFamily) {
         if (addrFamily == null) {
             throw new NullPointerException("addrFamily");
         }
 
         switch (addrFamily) {
-            case AF_UNSPEC:
-                if (address != null) {
-                    throw new HAProxyProtocolException("unable to validate an AF_UNSPEC address: " + address);
-                }
-                return;
-            case AF_UNIX:
-                return;
+        case AF_UNSPEC:
+            if (address != null) {
+                throw new HAProxyProtocolException("unable to validate an AF_UNSPEC address: " + address);
+            }
+            return;
+        case AF_UNIX:
+            return;
```

Comment:
@normanmaurer: Sure. The changes were applied after I used the Netty code template for IntelliJ since the original formatting seems to have an alternative coding style. Would be no problem to revert the formatting changes